src/factories/Container.ts

```diff
@@ -0,0 +1,93 @@
+module n3Charts.Factory {
+  'use strict';
+
+  export interface IDimension {
+    width: number;
+    height: number;
+    innerWidth: number;
```

Comment:
well, I haven't thought about this :D
I thought it would be nice to just grab the container factory and use the dim property directly